python/plugins/processing/gui/MultipleInputPanel.py

```diff
@@ -60,7 +60,7 @@ def setSelectedItems(self, selected):
         # No checking is performed!
         self.selectedoptions = selected
         self.leText.setText(
-            self.tr('%d elements selected') % len(self.selectedoptions))
+            self.tr('%d elements selected') % self.selectedoptions)
```

Comment:
I'm afraid this is incorrect fix, as `self.selectedoptions` in the code defined as list